app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(files_to_copy_path: str) -> None:
+    files = files_to_copy_path.split()
+    command = files[0]
+    if command == "cp":
+        if files[1] == files[2]:
```

Comment:
The indexing of the `files` list is incorrect. Ensure that `files[1]` is the source file and `files[2]` is the destination file. This check should ensure that the source and destination files are not the same.